src/tables.ts

```diff
@@ -1308,7 +1305,7 @@ export const Tables = {
             flag: false,
             rest: true,
             position: 'B/4',
-            dot_shiftY: 1.5,
+            dot_shiftY: -2.5,
```

Comment:
This change does not affect the output of the tests?

Reply:
We never tested 128th rests before. When I added a 128th rest to the test cases, I noticed the dot was in the wrong place.

Reply:
Good catch!